.env

```diff
@@ -0,0 +1,2 @@
+export MY_NAME=postgres
+export MY_PASSWORD=12345678
```

Comment:
- [ ] I would not recommend pushing important files such as `.env` in Github. Instead, you can mention the step that what file and variable one need to make to run the application. Please remove this file and put this in `.gitignore` file.